mumd/src/audio.rs

```diff
@@ -375,15 +238,7 @@ impl Audio {
 
     pub fn play_effect(&self, effect: NotificationEvents) {
         let samples = self.sounds.get(&effect).unwrap();
-
-        let mut play_sounds = self.play_sounds.lock().unwrap();
-
-        for (val, e) in play_sounds.iter_mut().zip(samples.iter()) {
-            *val = val.saturating_add(*e);
-        }
-
-        let l = play_sounds.len();
-        play_sounds.extend(samples.iter().skip(l));
+        self.client_streams.lock().unwrap().extend(None, samples);
```

Comment:
With the new ClientStreams, the key could Some(id) for user streams, and None for anything else, in this case audio effects.